faq-bundle/src/Resources/contao/dca/tl_faq.php

```diff
@@ -486,6 +523,18 @@ public function checkPermission()
 		}
 	}
 
+	public function checkSerpPreview()
+	{
+		$objFaqCategory = FaqCategoryModel::findByPk(CURRENT_ID);
```

Comment:
Not sure if we can/should use the model here. @aschempp Did this not cause problems in some cases?

Reply:
Changed with 1e3daf6